scipy/sparse/_bsr.py

```diff
@@ -703,6 +703,7 @@ class bsr_array(_bsr_base, sparray):
     -----
     Sparse arrays can be used in arithmetic operations: they support
     addition, subtraction, multiplication, division, and matrix power.
+    Block Sparse Row format sparse arrays do not support slicing.
```

Comment:
Can we move this to the bottom of the Notes section with a heading of `**Limitations**`?

Similar changes to `bsr_matrix`.